index.js

```diff
@@ -8,6 +8,7 @@ module.exports = {
 
     app.import('vendor/node-webkit/shim.js', { prepend: true });
     app.import({ development: 'vendor/node-webkit/reload.js' });
+    app.import({ development: '../../node_modules/ember-cli/node_modules/testem/node_modules/socket.io/node_modules/socket.io-client/socket.io.js' });
```

Comment:
> I thought that the testem wouldn't start the launcher until the server is already up but seems like the timing isn't guaranteed.

How are you determining that? The launchers shouldn't be initialized until the server is ready according to [this](https://github.com/airportyh/testem/blob/19a7f46bfc631d2cadbdc892a090a1d348a601c3/lib/dev/index.js#L88-L106), unless I'm reading that wrong.